components/global/product_switcher.tsx

```diff
@@ -67,7 +67,8 @@ const MenuItem = styled(Link)`
 `;
 
 const StyledIcon = styled(Icon)`
-    color: var(--sidebar-bg);
+    color: var(--button-bg);
+    font-size: 24px;
```

Comment:
Ah yes, the icon sizes are somewhat deceiving because the size of the icon is for the container and not specifically the font-size of the icon itself, which is usually a bit bigger to compensate for the extra padding the font adds around the glyph.

In this case, the font-size for the newer 24px icon is actually 28.8px is Figma. 🤷‍♂️

We absolutely should add the new size to the icon component, but in the mean time, this should probably be more like:

```suggestion
    ::before {
        font-size: 28.8px;
        line-height: 28.8px;
    }
```

Reply:
I will open a PR in compass components to add the icon size as Michael said this is a part of the documentation so we can use it directly.